ci/prow/Makefile

```diff
@@ -23,7 +23,7 @@ get-cluster-credentials:
 	gcloud container clusters get-credentials "$(CLUSTER)" --project="$(PROJECT)" --zone="$(ZONE)"
 
 config:
-	go run make_config.go config_knative.yaml > config.yaml
+	go run make_config.go --generate-testgrid-config=false --prow-config-output="config.yaml" config_knative.yaml
```

Comment:
This should also generate the Testgrid config, so both configs are kept in sync.

Reply:
Add Testgrid config generation in prow's Makefile.